src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/boss_pyroguard_emberseer.cpp

```diff
@@ -437,6 +366,7 @@ class npc_blackhand_incarcerator : public CreatureScript
 
         private:
             EventMap _events;
+            InstanceScript* _instance;
```

Comment:
where is this member field used ?

Reply:
Good point, forgot to remove it from a previous version where it was used.